ocs_ci/ocs/openshift_lightspeed.py

```
@@ -0,0 +1,568 @@
+                        "Authorization": f"Bearer {api_token}",
+                        "Content-Type": "application/json",
+                    },
+                    json={
+                        "model": model_name,
+                        "messages": [{"role": "user", "content": "ping"}],
+                        "max_tokens": 1,
+                    },
+                    verify=self.verify_tls,
+                    timeout=(10, 30),
+                )
+                return resp.status_code < 500
```

Ruff (0.16.3)

[warning] Consider moving this statement to an `else` block

(TRY300)